frontend/src/v5/ui/controls/inputs/inputController.component.tsx

```diff
@@ -66,7 +67,7 @@ export const InputController = <T,>({
 					field.onBlur();
 					onBlur?.();
 				}}
-				inputRef={ref}
+				inputRef={inputRef || ref}
```

Comment:
What happened here?
I think ref  can only be passed using fowardRef (it doesnt show as a prop)

Reply:
https://mui.com/material-ui/api/input/

inputRef is a prop accepted by MUI input. It allows to pass a ref down to the actual input component without using `forwardRef` several times across the children before reaching the input.

Here you can see that the inputRef prop reaches `InputBase` and after that it is treated like an actual ref, so no longer visible in children as a prop
![image](https://user-images.githubusercontent.com/43547703/220885072-2a33f946-49b1-40dd-af37-17a23acbf09d.png)

Reply:
I think we should have one way of using it either ref or inputRef, but not both.

Reply:
The ref is passed an actual ref now